<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Tasks</class>
 <widget class="QMainWindow" name="Tasks">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>761</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Tasks</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLineEdit" name="tasks_text">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>110</y>
      <width>481</width>
      <height>111</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>25</pointsize>
     </font>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>10</y>
      <width>250</width>
      <height>40</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>38</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Текст задачи</string>
    </property>
   </widget>
   <widget class="QLabel" name="first_variable">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>240</y>
      <width>60</width>
      <height>35</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>25</pointsize>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>240</y>
      <width>51</width>
      <height>35</height>
     </rect>
    </property>
    <property name="text">
     <string>min</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>240</y>
      <width>60</width>
      <height>35</height>
     </rect>
    </property>
    <property name="text">
     <string>max</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="first_whole">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>280</y>
      <width>100</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Целое</string>
    </property>
    <attribute name="buttonGroup">
     <string notr="true">buttonGroup</string>
    </attribute>
   </widget>
   <widget class="QRadioButton" name="first_material">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>280</y>
      <width>121</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Вещественное</string>
    </property>
    <attribute name="buttonGroup">
     <string notr="true">buttonGroup</string>
    </attribute>
   </widget>
   <widget class="QPushButton" name="back">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>41</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;--</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Run">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>220</y>
      <width>151</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Найти переменные</string>
    </property>
   </widget>
   <widget class="QLabel" name="fourth_variable_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>550</y>
      <width>211</width>
      <height>35</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>25</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Кол-во вариантов</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="spinBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>590</y>
      <width>91</width>
      <height>24</height>
     </rect>
    </property>
    <property name="minimum">
     <number>1</number>
    </property>
   </widget>
   <widget class="QLabel" name="fourth_variable_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>620</y>
      <width>251</width>
      <height>35</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>25</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Формула для задачи</string>
    </property>
   </widget>
   <widget class="QLabel" name="fourth_variable_4">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>670</y>
      <width>371</width>
      <height>35</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="text">
     <string>*Используйте те же буквы, что и в условии</string>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="first_min">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>240</y>
      <width>68</width>
      <height>24</height>
     </rect>
    </property>
    <property name="minimum">
     <double>-9999.000000000000000</double>
    </property>
    <property name="maximum">
     <double>9999.989999999999782</double>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="first_max">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>240</y>
      <width>68</width>
      <height>24</height>
     </rect>
    </property>
    <property name="minimum">
     <double>-9999.000000000000000</double>
    </property>
    <property name="maximum">
     <double>9999.989999999999782</double>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>60</y>
      <width>451</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Введите тест задачи, нужные данные запиши в этом знаке &amp;&amp;</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>86</y>
      <width>121</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Пример: &amp;N&amp;</string>
    </property>
   </widget>
   <widget class="QPushButton" name="save">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>560</y>
      <width>161</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Сгенерировать</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="formula">
    <property name="geometry">
     <rect>
      <x>262</x>
      <y>610</y>
      <width>321</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
     </font>
    </property>
   </widget>
   <widget class="QPushButton" name="file">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>660</y>
      <width>191</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Сохранить</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>600</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
 <buttongroups>
  <buttongroup name="buttonGroup"/>
 </buttongroups>
</ui>
